girder/utility/server.py

```diff
@@ -50,7 +68,9 @@ def configureServer(test=False, plugins=None, curConfig=None):
         '/': {
             'request.dispatch': cherrypy.dispatch.MethodDispatcher(),
             'request.show_tracebacks': test,
-            'request.methods_with_bodies': ('POST', 'PUT', 'PATCH')
+            'request.methods_with_bodies': ('POST', 'PUT', 'PATCH'),
+            'response.headers.server': 'Girder ' + API_VERSION,
```

Comment:
Another option would be to send an empty string for this header value.  If we want to generally start reducing the number of places this is exposed.